pkg/keys/fulcio.go

```diff
@@ -0,0 +1,280 @@
+package keys
+
+import (
+	"context"
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/coreos/go-oidc"
+	httptransport "github.com/go-openapi/runtime/client"
+	fulcioClient "github.com/sigstore/fulcio/pkg/client"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/sigstore/fulcio/pkg/generated/client/operations"
+	"github.com/sigstore/fulcio/pkg/generated/models"
+	"github.com/sigstore/sigstore/pkg/oauthflow"
+	"github.com/theupdateframework/go-tuf/data"
+)
+
+func init() {
```

Comment:
Init methods are hard to test https://peter.bourgon.org/blog/2017/06/09/theory-of-modern-go.html

Reply:
I think that this is OK. This init function is registering this particular key type in to the registered key map that go-tuf can understand. Unit tests for these individual key types exist independently. 

The alternative in that post is to define them in a main function -- that could be the case, but people also use go-tuf as a library.